src/components/Form/index.tsx

```diff
@@ -199,114 +257,139 @@ export const Form = ({
               error={Boolean(errors.claim)}
               helperText={errors.claim?.message}
             >
-              {inputOptions.claim.map((i: string) => (
-                <MenuItem value={i} key={i}>
-                  {i}
+              {inputOptions.claim.map((claimText: string, index: number) => (
+                <MenuItem value={claimText} key={claimText}>
+                  <Tooltip key={claimText} title={tooltips.claim[index]} placement='right' arrow>
+                    <Box sx={{ width: '100%', height: '100%' }}>{claimText}</Box>
+                  </Tooltip>
                 </MenuItem>
               ))}
             </TextField>
-            <TextField
-              select
-              label='How Known'
-              {...register('howKnown')}
-              sx={{ ml: 1, mr: 1, width: '22ch' }}
-              margin='dense'
-              variant='outlined'
-              fullWidth
+            <Tooltip title='The method or source of the claim ' placement='right' arrow>
+              <TextField
+                select
+                label='How Known'
+                {...register('howKnown')}
+                sx={{ ml: 1, mr: 1, width: '22ch' }}
+                margin='dense'
+                variant='outlined'
+                fullWidth
+              >
+                {inputOptions.howKnown.map((howKnownText: string, index: number) => (
+                  <MenuItem value={howKnownText} key={howKnownText}>
+                    <Tooltip key={howKnownText} title={tooltips.howKnown[index]} placement='right' arrow>
```

Comment:
Do `Tooltip` need the `key` prop? If not remove it plz.

Reply:
I removed it, thanks